apps/web/lib/services/client.service.ts

```
@@ -0,0 +1,141 @@
+      console.error(`Error fetching client ${id}:`, error);
+      return null;
+    }
+  }
+
+  /**
+   * Get a client by ID and convert to ClientInfo format
+   * @param id Client ID
+   * @returns ClientInfo formatted data
+   */
+  static async getClientInfoById(id: string): Promise<Client | null> {
+    const client = await this.getClientById(id);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)